WordPress/Classes/ViewRelated/Post/EditPostViewController.swift

```diff
@@ -126,19 +126,12 @@ class EditPostViewController: UIViewController {
     }
 
     private func showEditor(_ editor: EditorViewController) {
-        editor.onClose = { [weak self, weak editor] changesSaved in
-            guard let strongSelf = self else {
+        editor.onClose = { [weak self, weak editor] in
+            guard let self else {
                 editor?.dismiss(animated: true) {}
                 return
             }
-
-            // NOTE:
-            // We need to grab the latest Post Reference, since it may have changed (ie. revision / user picked a
-            // new blog).
-            if changesSaved {
-                strongSelf.post = editor?.post as? Post
```

Comment:
This is no longer how it works starting from 23.9. After publishing `editor?.post as? Post` points to a _revision_ that gets deleted during publishing. `self.post` is sufficient now.